README.md

```diff
@@ -337,3 +338,17 @@ const contributor: TraceServerContributor = {
 
 importedApi.addTraceServerContributor(contributor);
 ```
+
+### Remote Support
+VSCode Trace Extension is compatible with the use of remote-ssh.  If you are using remote-ssh, you can simply install Trace Viewer on your remote machine and begin using the extension.
```

Comment:
I had multiple comments to update. Instead of asking for changes per sentence, I provided below an updated version of this chapter as replacement. 

Please note that the extension is now deployed on the VSCode marketplace and OpenVSX marketplace. The official name of the extension on the marketplaces is `Trace Viewer for VSCode`. 

```
The `Trace Viewer for VSCode` extension is compatible with the use of [remote-ssh](https://code.visualstudio.com/docs/remote/ssh).  If you are using `remote-ssh`, you can simply install `Trace Viewer for VSCode` extension on your remote machine and begin using the extension. 

The remote support works by forwarding the Trace Server's operating port from the client machine to the remote machine. The port is automatically forwarded on Trace Server startup.

Forwarded ports can be seen in the 'Ports' view of the remote VsCode.  To open the `Ports` view use menu `View -> Open view... -> Ports`.  You should see the forwarded Trace Server port in the `Ports` view, as shown below:

![ports-tab](vscode-ports-tab)

Make sure that there is no Trace Server running on your local host. If the `Trace Viewer for VSCode` is unresponsive, stop the port forwarding by pressing the 'Stop Port Fowarding (Delete)' of the trace server port and restart remote VSCode.

If you are a developers of the `Trace Viewer for VsCode` and want to modify and test the extension, you can [package it as a VSCode extension (.vsix)](#package-as-a-vscode-extension-vsix), upload the `VSIX` to the remote host and install the extension using the `Install from VSIX...` view menu of the `Extensions` view.
```